charts/vela-core/templates/defwithtemplate/storage.yaml

```diff
@@ -154,7 +154,7 @@ spec:
         					}
 
         					if v.resources.requests.storage == _|_ {
-        						resources: requests: storage: "1Gi"
+        						resources: requests: storage: "8Gi"
```

Comment:
why do we request the 8Gi instead of 10Gi or 20Gi？

Reply:
Refer to Helm Chart, the usual setting is 8Gi.